src/shell-interface.h

```diff
@@ -50,6 +50,10 @@ struct ShellExternalInterface : ModuleInstance::ExternalInterface {
 
    public:
     Memory() {}
+    char* rawbuffer(unsigned offset, unsigned length) {
+      assert(size() <= (offset + length), "memory: requested raw buffer is too short");
```

Comment:
Do we want to use heraAssert macro here, or settle for plain assert?

Reply:
It is a different file where `heraAssert` is not available.

Reply:
Should it not be moved to a place where it is more accessible, then?
I think that unless heraAssert is specifically intended for EEI, it should be put in a header of project-wide visibility so that any objects that use it do not have to include eei.h.

Reply:
After #187 it can use those exceptions.